src/content/en/tools/lighthouse/audits/descriptive-link-text.md

```diff
@@ -0,0 +1,48 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Links Do Not Have Descriptive Text" Lighthouse audit.
+
+{# wf_updated_on: 2018-02-22 #}
+{# wf_published_on: 2018-02-22 #}
+{# wf_blink_components: N/A #}
+
+# Links Do Not Have Descriptive Text  {: .page-title }
```

Comment:
How much do you think these docs should go into the SEO benefits of [doing the thing the audit checks for]? Linking to relevant SEO docs is good too.

Reply:
I thought about that, but I haven't really seen anything else on it, other than "it helps search engines understand your content"

We could link to webmaster blog, to give it authority, but the content is pretty much duplicated

Reply:
Here's one: https://support.google.com/webmasters/answer/7451184#optimize (scroll down to "Write good link text" section, no closer anchor to jump to... grr)